Sources/Core/Other/PasteboardMonitor.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  PasteboardMonitor.swift
+//  XestiMonitors
+//
+//  Created by Paul nyondo on 2018-03-15.
+//
+//  © 2016 J. G. Pusey (see LICENSE.md)
+//
+
+#if os(iOS)
+    import  UIKit
+    ///
+    /// A `PasteboardMonitor` instance monitors a pasteboard for changes to
+    /// its state.
+    ///
+    public class PasteboardMonitor: BaseNotificationMonitor {
+        
+        ///
+        /// Encapsulates changes to the contents of the pasteboard.
+        ///
+        
+        public enum Event {
+            ///
+            ///  Contents of the pasteboard object have changed.
+            ///
+            case didChange(Info)
+            ///
+            ///  Contents of the pasteboard object have been removed.
+            ///
+            case didRemove(Info)
+            
+        }
+        
+        public struct Info {
```

Comment:
This needs a doc comment. (I aim for 100% documentation coverage.) See `KeyboardMonitor` for an example.